packages/pdf/src/RPA/PDF/keywords/model.py

```diff
@@ -478,8 +479,8 @@ class ModelKeywords(LibraryContext):
     @keyword
     def convert(
         self,
-        source_path: str = None,
-        trim: bool = True,
+        source_path: Optional[str] = None,
+        trim: Optional[bool] = True,
```

Comment:
Nit: Actually this should stay: `trim: bool = True,` as you usually pass `True`/`False` to the param, not `None`.